packages/cli/src/lib/axe-test-urls.test.ts

```diff
@@ -67,25 +67,25 @@ describe('testPages', function () {
   });
 
   it('injects axe into the page', async () => {
-    const scripts = [];
+    const scripts: string[] = [];
     config.axeSource = 'axe="hi, I am axe"';
-    mockDriver.executeScript = async script => {
+    mockDriver.executeScript = async (script: string) => {
       scripts.push(script);
       return script;
     };
 
-    await testPages(['http://foo'], config, {});
+    await testPages(['http://foo'], config, {} as any);
     assert.include(scripts[0].toString(), config.axeSource);
   });
 
   it('runs axe once the page is loaded', async () => {
-    const asyncScripts = [];
-    mockDriver.executeAsyncScript = async script => {
+    const asyncScripts: string[] = [];
+    mockDriver.executeAsyncScript = async (script: string) => {
       asyncScripts.push(script);
       return script;
     };
 
-    await testPages(['http://foo'], config, {});
+    await testPages(['http://foo'], config, {} as any);
```

Comment:
looks like the third argument can be optional, lets make it `events?: EventResponse` instead of passing an empty object.